ide-common/src/main/kotlin/org/digma/intellij/plugin/auth/AuthManager.kt

```diff
@@ -69,38 +70,102 @@ class AuthManager {
     //withAuth should not be called concurrently by multiple threads. it is called only from AnalyticsService.replaceClient.
     //AnalyticsService.replaceClient is called on startup and when a relevant settings change and is synchronized.
     //the analyticsProvider here will usually be a non proxied RestAnalyticsProvider
-    fun withAuth(analyticsProvider: RestAnalyticsProvider, url: String): AnalyticsProvider {
+    fun withAuth(analyticsProvider: RestAnalyticsProvider): AnalyticsProvider {
 
```

Comment:
you can keep the analyticsProvider here as class member and reuse it later for login